server/routes/index.ts

```diff
@@ -536,5 +539,15 @@ export default function routes(services: Services): Router {
     courtCaseRoutes.getDraftConfirmationPage,
   )
 
+  get(
+    '/person/:nomsId/:addOrEditCourtCase/:courtCaseReference/:addOrEditCourtAppearance/:appearanceReference/sentencing/overall-sentence-length',
```

Comment:
have hard coded 'sentencing' in the url path here as these routes only occur for sentencing. I think for other routes where where this is overlap we still split out the routes but have :warrantType as a uerl param... (BUT I didnt do that here, because these routes dont apply to remand at all)